assets/scripts/util/fetch_blocking.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * Create blocking fetch requests.
+ * Display a blocking shield when a request is in progress.
+ * Only one of these should be active at any time and
+ * no other fetch / xhr / ajax should exist when they do.
+ */
+import $ from 'jquery'
+import { showBlockingShield, hideBlockingShield, darkenBlockingShield } from '../app/blocking_shield'
+
+let blockingAjaxRequest
+let blockingAjaxRequestDoneFunc
+let blockingAjaxRequestCancelFunc
+
+// This needs to be exported to window for streets/xhr (temp?)
+export let blockingAjaxRequestInProgress = false
```

Comment:
I think ES6 `export` might work differently?

See here: http://www.2ality.com/2015/07/es6-module-exports.html

`blockingAjaxRequestInProgress` should be bound to the module's values. Let me see if I can whip up something that proves this.